Auto Translation Bug

OS: Mac OS
Node: 8.4.0
Hi, This is my xliffmergeOptions, my default language is 'zh-CN', when I try to translate zh-CN to en with auto translation, I got the error message below:

`{
  "xliffmergeOptions": {
    "srcDir": "src/i18n",
    "genDir": "src/i18n",
    "i18nFile": "messages.xlf",
    "i18nFormat": "xlf",
    "encoding": "UTF-8",
    "defaultLanguage": "zh-CN",
    "languages": ["en", "ja"],
    "removeUnusedIds": true,
    "supportNgxTranslate": true,
    "useSourceAsTarget": false,
    "autotranslate": ["en"],
    "apikey": "***",
    "apikeyfile": "",
    "verbose": false,
    "quiet": false
  }
}`

`ERROR: autotranslate language "en" cannot be translated, because it is the source language
npm ERR! code ELIFECYCLE
npm ERR! errno 255
npm ERR! service-engine@0.0.0 translate:file: `ng xi18n --output-path src/i18n --locale zh-CN && xliffmerge --profile xliffmerge.json en ja`
npm ERR! Exit status 255
npm ERR! 
npm ERR! Failed at the service-engine@0.0.0 translate:file script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/liuxinqiang/.npm/_logs/2017-08-29T09_08_49_961Z-debug.log`

--- a/Changelog.md
+++ b/Changelog.md
@@ -1,3 +1,10 @@
+<a name="0.8.5"></a>
+# [0.8.5](https://github.com/martinroob/ngx-i18nsupport/compare/v0.8.4...v0.8.5) (2017-08-29)
+
+### Bug Fixes
+
+* **autotranslate:** Autotranslate parameter is checked against first given language instead of default language ([#52](https://github.com/martinroob/ngx-i18nsupport/issues/52)).
+
 <a name="0.8.4"></a>
 # [0.8.4](https://github.com/martinroob/ngx-i18nsupport/compare/v0.8.3...v0.8.4) (2017-08-22)
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-i18nsupport",
-  "version": "0.8.4",
+  "version": "0.8.5",
   "description": "Some tooling to be used with the Angular 2 i18n workflow",
   "main": "index.js",
   "module": "./src",

--- a/src/xliffmerge/xliff-merge-parameters.ts
+++ b/src/xliffmerge/xliff-merge-parameters.ts
@@ -214,7 +214,7 @@ export class XliffMergeParameters {
             if (this.languages().indexOf(lang) < 0) {
                 this.errorsFound.push(new XliffMergeError('autotranslate language "' + lang + '" is not in list of languages'));
             }
-            if (this.languages().length > 0 && lang === this.languages()[0]) {
+            if (lang === this.defaultLanguage()) {
                 this.errorsFound.push(new XliffMergeError('autotranslate language "' + lang + '" cannot be translated, because it is the source language'));
             }
         });

--- a/src/xliffmerge/xliff-merge.spec.ts
+++ b/src/xliffmerge/xliff-merge.spec.ts
@@ -248,6 +248,23 @@ describe('XliffMerge test spec', () => {
             done();
         });
 
+        it('should not output error ".. because it is the source language" when autotranslate language is not set to default language (issue #52)', (done) => {
+            let ws: WriterToString = new WriterToString();
+            let commandOut = new CommandOutput(ws);
+            let profileContent: IConfigFile = {
+                xliffmergeOptions: {
+                    defaultLanguage: 'zh-CN',
+                    languages: ['en', 'ja'],
+                    autotranslate: ['en', 'ja'],
+                }
+            };
+            let xliffMergeCmd = XliffMerge.createFromOptions(commandOut, {}, profileContent);
+            xliffMergeCmd.run();
+            expect(ws.writtenData()).toContain('ERROR');
+            expect(ws.writtenData()).not.toContain('autotranslate language "en" cannot be translated, because it is the source language');
+            done();
+        });
+
         it('should accept i18n format xlf', (done) => {
             let ws: WriterToString = new WriterToString();
             let commandOut = new CommandOutput(ws);

